dfinity/nns-dapp
# GIX-2080: Set ENABLE_MY_TOKENS true by default

# Motivation

Enable ENABLE_MY_TOKENS in mainnet.

# Changes

* Set `ENABLE_MY_TOKENS` to `true` in dfx.json

# Tests

* Change the expected title `"My ICP Tokens / NNS Dapp"` to `"My Tokens / NNS Dapp"` for e2e tests not starting in `/accounts`. This wasn't done before because we were waiting for the title to set the flag.
* Change the value of `ENABLE_MY_TOKENS` in the config tests.

# Todos
